### ISSUE TITLE
Disabling Biomes? [1.12.2, Technic]

### ISSUE THREAD

**Issue by SPYROHAWK** [pale-imitations/Magic_Kingdoms#44]
Hey, I had a quick question / issue I can't seem to find documentation for. In a pack I am working on we want to disable all Magic Kingdoms / SOM biomes from generating in the overworld. We have a whole thing going on with custom world gen that doesn't work well with modded biomes, but we have Mystcraft which can be used to access them for whatever structures and resources are needed from there.

This issue is we can't exactly figure out *how*. Well, we can but it doesn't work. Let me explain.

In the config file, there's configs for the biome spawn rates. However, we don't know if the chances are X% or 1/X% (so bigger number higher chance or bigger number lower chance).

Now, the way we are making our modpack is we start it on Twitch for easy adding and cutting mods, then since Twitch doesn't work for some people in our group, we move to Technic. Now, on Twitch the configs seem to be working. We set the configs to 0% and the biomes don't spawn. Fantastic.

But somehow moving to Technic causes a whole host of issues. We're having some WAILA render issues, some potential issues with some other mod configs such as Mo'Villages, and our SOM issue. The biomes do indeed keep generating in the overworld, no matter what we set the configs to. Note the only mod difference between the two versions is the Technic version has Optifine while the Twitch does not.

My personal working theory is that for some reason Technic is not listening to any edits we make to the config files, which is why the biome changing isn't working. The other option is that the configs are not working regardless, and we just got really lucky on the Twitch testing. The other OTHER option is something with setting the value to 0% causes an issue with it trying to compute a 1/X% or something, but setting it to a super small decimal or 9999999 also doesn't work (although once again, that assumption has flaws, like if 99999 is super high chance and the decimal we are trying is getting rounded to 0 and causing problems again).

Bottom line, biomes are spawning and we don't know why or how to get them to stop.
